src/nvim/fileio.c

```diff
@@ -5341,10 +5356,10 @@ static int au_find_group(char_u *name)
   return AUGROUP_ERROR;
 }
 
-/*
- * Return TRUE if augroup "name" exists.
- */
-int au_has_group(char_u *name)
+/// Return true if augroup "name" exists.
+///
+/// @param name augroup name
+bool au_has_group(char_u *name) FUNC_ATTR_PURE FUNC_ATTR_WARN_UNUSED_RESULT
```

Comment:
Missed this: both au_has_group and au_find_group may have `const char_u *` arguments.